configure

```diff
@@ -5328,9 +5329,10 @@ if test "$_vpx" = auto ; then
 
 	if test "$_pkg_config" = "yes" && $_pkgconfig --exists vpx; then
 		append_var VPX_LIBS "`$_pkgconfig --libs vpx`"
+		append_var VPX_STATIC_LIBS "`$_pkgconfig --static --libs vpx`"
 		append_var VPX_CFLAGS "`$_pkgconfig --cflags vpx`"
 	else
-		append_var VPX_LIBS "-lvpx"
+		append_var VPX_LIBS "-lvpx -lpthread"
```

Comment:
pthread might not exists on some platforms

Reply:
Indeed, but if libvpx has been compiled with multithreading (it's the default) and is static, this won't work either.
I will remove it though.